packages/date-time-utilities/package.json

```diff
@@ -24,7 +24,6 @@
     "update-api": "just-scripts update-api"
   },
   "devDependencies": {
-    "@fluentui/eslint-plugin": "^0.54.1",
```

Comment:
This needs to stay too--the fact that this package doesn't have eslintrc is wrong.